### ISSUE TITLE
Stars card in Greed Mode

### ISSUE THREAD

**Issue by Lonrite** [UnderscoreKilburn/repentance-issues#2000]
that card is a laughing stock in here, it should stop teleporting you to silver / white treasures which are free to enter anyways.

also would be neat if planetariums would be able to spawn to the right side of any of the treasures - aka red key / blue key rooms

**Comment by ZMBanshee** [comment]
I agree that it should prioritize the gold room, but you can guarantee that by visiting the silver one first.

**Comment by Lonrite** [comment]
> I agree that it should prioritize the gold room, but you can guarantee that by visiting the silver one first.

oh ty, didn't know that

for some reason I assumed it would be similar to how stars card works in vanilla mode with a planetarium in-tact.

Like, even if you already explore planetarium on a given floor, yet never enter the treasure - it'd teleport you to planetarium no matter what
